test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -670,18 +670,18 @@ def init_servers(cls, init_server_func=None, skip=None):
         if init_server_func is None:
             init_server_func = cls.init_server
         if 'servers' in cls.conf:
-            if 'comms' not in cls.conf:
+            if 'comms' not in cls.conf and 'comm' not in cls.conf:
                 cls.conf['comms'] = cls.conf['servers']
-            if 'schedulers' not in cls.conf:
+            if 'schedulers' not in cls.conf and 'scheduler' not in cls.conf:
                 cls.conf['schedulers'] = cls.conf['servers']
-            if 'moms' not in cls.conf:
+            if 'moms' not in cls.conf and 'mom' not in cls.conf:
                 cls.conf['moms'] = cls.conf['servers']
         if 'server' in cls.conf:
-            if 'comm' not in cls.conf:
+            if 'comms' not in cls.conf and 'comm' not in cls.conf:
                 cls.conf['comm'] = cls.conf['server']
-            if 'scheduler' not in cls.conf:
+            if 'schedulers' not in cls.conf and 'scheduler' not in cls.conf:
                 cls.conf['scheduler'] = cls.conf['server']
-            if 'mom' not in cls.conf:
+            if 'moms' not in cls.conf and 'mom' not in cls.conf:
```

Comment:
we can simplify the code to something like -
if 'server' in cls.conf:
        name = 'server'               #Use any identifier for the variable - I am using name just as an example.
elif 'servers' in cls.conf:
        name = 'servers'
if 'comms' not in cls.conf and 'comm' not in cls.conf:
                cls.conf['comm'] = cls.conf[name]
if 'schedulers' not in cls.conf and 'scheduler' not in cls.conf:
                cls.conf['scheduler'] = cls.conf[name]
if 'moms' not in cls.conf and 'mom' not in cls.conf:
                cls.conf['mom'] = cls.conf[name]

Reply:
@prakashcv13 I have addressed your review comments. Please have a look